config/constants.js

```diff
@@ -1,8 +1,13 @@
+'use strict';
+
 module.exports = {
-  AWS : {
-    BUCKET_NAME : 'my_bucket',
+  AWS: {
+    BUCKET_NAME: 'my_bucket',
     ACCESS_KEY_ID: 'ACCESS_KEY_ID',
     SECRET_ACCESS_KEY: 'SECRET_ACCESS_KEY',
     ENDPOINT: 'http://localhost:4567'
+  },
```

Comment:
All these variables can be moved to environment variables so that they can be varied depending upon the environment- staging, production, development.